File: src/main/java/gargoyle/ct/ex/CTException.java
package gargoyle.ct.ex;

public class CTException extends RuntimeException {
    private static final long serialVersionUID = -4669219915320326863L;

    public CTException(String message) {
        super(message);
    }

    public CTException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: src/main/kotlin/gargoyle/ct/ex/CTException.kt
package gargoyle.ct.ex

import java.io.Serial

class CTException : RuntimeException {
    constructor(message: String?) : super(message)
    constructor(message: String?, cause: Throwable?) : super(message, cause)

    companion object {
        @Serial
        private val serialVersionUID = -4669219915320326863L
    }
}
